### PR TITLE
Revert remaining changes from PR#14004

### PR DESCRIPTION
**Pull request by biyeun**
https://github.com/dimagi/commcare-hq/pull/14004

Causing issues with form loading in firefox...

@benrudolph 
@wpride

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 1]
```diff
@@ -427,12 +417,6 @@ FormplayerFrontend.on('refreshApplication', function(appId) {
         tfLoadingComplete(true);
     }).done(function() {
         tfLoadingComplete();
-        if (!_.isUndefined(resp.responseJSON.tree)) {
-            FormplayerFrontend.trigger('startForm', resp.responseJSON);
-        } else {
-            FormplayerFrontend.trigger('navigateHome');
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: this is still needed

**Review by benrudolph — CHANGES_REQUESTED** [review]

**Comment by benrudolph** [comment]
@wpride it was causing issues on firefox so we decided to revert until we have updates. when that gets in we can then worry about how to fix these issues

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js (hunk 1 above)
Comment (biyeun) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: thanks for the catch. putting this back

**Comment by biyeun** [comment]
@benrudolph responded

**Review by benrudolph — APPROVED** [review]

**Comment by biyeun** [comment]
@benrudolph ✅

**Pull request merged**